db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_329_185_029) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.string 'Text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id'
+    t.bigint 'posts_id', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['posts_id'], name: 'index_comments_on_posts_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'users_id', null: false
+    t.bigint 'posts_id', null: false
+    t.index ['posts_id'], name: 'index_likes_on_posts_id'
+    t.index ['users_id'], name: 'index_likes_on_users_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.string 'text'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id'
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.string 'Bio'
+    t.integer 'posts_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
+
+  add_foreign_key 'comments', 'posts', column: 'posts_id'
+  add_foreign_key 'comments', 'users', column: 'author_id'
+  add_foreign_key 'likes', 'posts', column: 'posts_id'
+  add_foreign_key 'likes', 'users', column: 'users_id'
+  add_foreign_key 'posts', 'users', column: 'author_id'
```

Comment:
- Hi :clap: , After you've made all the changes I suggested above, you'd have dropped your database, create it again and run the migration, everything should work correctly with no errors.

`rails db:drop db:create db:migrate`

- Then, In your `schema.rb` file, you should see something like this below:


-should look something like this:

```sql
  add_foreign_key "comments", "posts"
  add_foreign_key "comments", "users"
  add_foreign_key "likes", "posts"
  add_foreign_key "likes", "users"
  add_foreign_key "posts", "users", column: "author_id"
```